cmd/zrepl/mgmt_conn.c

```diff
@@ -741,8 +909,10 @@ move_to_next_state(uzfs_mgmt_conn_t *conn)
 		if (vers != REPLICA_VERSION) {
 			fprintf(stderr, "invalid replica protocol version %d\n",
 			    vers);
-			rc = reply_error(conn, ZVOL_OP_STATUS_VERSION_MISMATCH,
```

Comment:
here, next state is supposed to be CS_CLOSE.. but, in reply_nodata, we are setting it to CS_INIT..
are we taking care of this in some other way?

Reply:
oh ok.. got to see the next line.. :) :+1: